addons/blue_secret_controller_test.go

```diff
@@ -1 +0,0 @@
-package addons
```

Comment:
Why is this file deleted? Were there no tests here?

Reply:
It was already empty. The tests were removed after refactors to the controllers. Will be added new